cpp/cgi_progs/newsletter_uploader.py

```diff
@@ -1,30 +1,42 @@
 #!/usr/bin/env python
 import cgi
 import cgitb
+import datetime
 import os
 
 
+print("Content-Type: text/html\n")
```

Comment:
I think the standard says that HTTP headers have to end in carriage-return line-feed pairs.